Sources/AWSSDKSwiftCore/Signers/V4.swift

```diff
@@ -264,6 +264,17 @@ extension Signers {
             }
             return result
         }
+        
+        /// Encode Query according to https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html
```

Comment:
Maybe add the relevant piece below that line too? (Up to you— it wasn't too hard to find but might be helpful)

```
`http://s3.amazonaws.com/examplebucket?acl`

The CanonicalQueryString in this case is as follows:
> `UriEncode("acl") + "=" + ""`
```